docs/site/src/utils/getopenrpcspecs.js

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+const axios = require('axios');
+const fs = require('fs');
+const path = require('path');
+
+// Download Mainnet OpenRPC spec
```

Comment:
I am not a JS expert, but I think it'd be better to have one function that is called with different parameters than copy pasting the code three times. 
```suggestion
function download_spec(branch) {
axios({
    method: "get",
    url: "https://raw.githubusercontent.com/MystenLabs/sui/${branch}/crates/sui-open-rpc/spec/openrpc.json",
    responseType: "blob"
}).then((res) => {

    if (fs.existsSync(path.join(__dirname, "../open-spec/${branch}/openrpc_backup.json"))){
        fs.unlink(path.join(__dirname, "../open-spec/${branch}/openrpc_backup.json"), (err) => {
            if (err) {
                return console.log(err);
            }
            console.log("Deleted ${branch} backup spec.")
        } )
    } else {
        console.log("Backup file for ${branch} does not exist.")
    }
    if (fs.existsSync(path.join(__dirname, "../open-spec/${branch}/openrpc.json"))){
        fs.renameSync(path.join(__dirname, "../open-spec/${branch}/openrpc.json"), path.join(__dirname, "../open-spec/${branch}/openrpc_backup.json"));
    }

    fs.writeFileSync(path.join(__dirname, "../open-spec/${branch}/openrpc.json"), res.data, 'utf8');
}).catch(err => {
    console.log("Error downloading ${branch} openrpc spec.");
    console.error(err);
})
// Download Mainnet OpenRPC spec
download_spec("mainnet");
// Download Devnet OpenRPC spec
download_spec("devnet");
// Download Testnet OpenRPC spec
download_spec("testnet");
```

Reply:
Thought the same but wanted to be verbose because it was small. Will update.

Reply:
Ultimately, it's up to you. Having a function will lower the chances of having bugs that might need to be fixed in 3 places.

Reply:
Yep. I updated. And the vercel preview worked so merging. Thanks!

Reply:
Awesome!!!